bin/check-event-json

```diff
@@ -1,15 +1,23 @@
 #!/usr/bin/env node
 
-var fs = require("fs");
-var filename = process.argv[2];
+var fs = require('fs');
+var file = process.argv[2];
 
-if (filename) {
+if(file !== undefined && file !== null) {
+  checkJson(file);
+} else {
+  for(var i of ['events.json', '2015/events.json', '2016/events.json']) {
```

Comment:
If we decide to migrate historic data it would grow a bit. We can move the 2015 and 2016 into a new directory `years`, and iterate over every year we find there